src/integrations/alerts/black-friday-sale-notification.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Yoast\WP\SEO\Integrations\Alerts;
+
+/**
+ * Black_Friday_Sale_Notification class.
+ */
+class Black_Friday_Sale_Notification extends Abstract_Dismissable_Alert {
```

Comment:
do we want to use sale here and not promotion?

Reply:
Changed `sale` to `promotion` everywhere